tests/apps/lims/test_apps_lims_orderform.py

```diff
@@ -358,6 +356,58 @@ def test_parsing_mip_balsamic_orderform(mip_balsamic_orderform):
     assert mother_sample.get('mother') is None
 
 
+def test_parsing_mip_rna_orderform(mip_rna_orderform):
```

Comment:
The problem with having 30 asserts in one test is that once the first one fails, we will not know if the others are failing as well. Does it make sense to split the asserts into their own test?

Reply:
I guess we could but if the first fails then youll be fixing and testing until your're through anyways

Reply:
yes. It's up to you, I'm just pointing it out.

Source: https://testandcode.com/65